deploy-java/action.yml

```diff
@@ -138,6 +138,19 @@ runs:
         jiraUsername: ${{ inputs.jiraUsername }}
         jiraPassword: ${{ inputs.jiraPassword }}
 
+    - name: On success notify okay to test
+      uses: actions/github-script@v6
+      if: ${{ (inputs.environment !='Production') }}
+      with:
+        github-token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        script: |
+          const result = await github.rest.actions.createWorkflowDispatch({
+            owner: 'cartoncloud',
+            repo: 'api-functional-tests',
+            workflow_id: 'run-tests.yml',
+            environment: ${{ inputs.environment }}
+          });
+
```

Comment:
So intent of this change is to automatically run an Api Functional tests suite on each successful deployment? If you deploy to QA1 it should run the suite there? 

When you cut a release it will automatically try to run the suite simultaneously against Staging01 and Test01.

Lastly what happens if you deploy to QA04? Api Functional Tests don't have a QA04 environment.

Reply:
This is really intended to be an "okay to test" event notification that may or may not be honoured for the target environment.  
Just like there's no production api functional tests, line 143 could be augment to list exclusions (really should be a property of whether it is supported or not, but that's not justified here.  Separately, I'd say that our environments should match each other as closely as possible.  I'll exclude QA04 when I obliterate the need for a mapping

Reply:
https://github.com/cartoncloud/devops-scripts/pull/1 to rename environments
https://github.com/cartoncloud/api-functional-tests/pull/100 for api-functional-test updates